app/controllers/admin/testimonies_controller.rb

```diff
@@ -11,9 +11,11 @@ def show
 
   def new
     @testimony = Testimony.new
+    @testimony.build_photo
```

Comment:
Is the photo here optional? If not, can we make it optional for now? because there is a request to not use photo for testimony for now, but might probably need it in the future so will be better if we can just set it optional. Otherwise, might be better to just remove photo from the testimony temporarily...

Reply:
Done

Reply:
Okok, nice, thankyou